entries/QUnit.config.xml

```diff
@@ -35,6 +35,11 @@
 				<p>Prior to version 1.16.0, this property was called <code>module</code>.</p>
 			</desc>
 		</property>
+		<property name="moduleId" type="Array" default="undefined">
+			<desc>
+				This property allows QUnit to run specific modules by a hashed string using their module name. You can specify one or multiple modules to run.
```

Comment:
I think this could be slightly clearer: "This property allows QUnit to run specific modules _identified by the hashed version of their module name_. You can specify one or multiple, _comma-separated_ modules to run."